### PR TITLE
Support Unicode-16 octant characters

### PR DESCRIPTION
**Pull request by eschnett**
They will look better than Braille. Once they are supported in fonts and terminal emulators...

### CODE REVIEW

**Review by rafaqz — APPROVED** [review]
Thanks, nice to have these

**Comment by adrhill** [comment]
Thanks a lot! Judging by your comment it’s normal that the references don’t render correctly on iOS Safari?

**Review by adrhill — APPROVED** [review]
Could you update the CHANGLOG.md and the docstring in `src/api.jl`, mentioning the terminal requirements?

**Comment by eschnett** [comment]
I updated the documentation.

File: README.md [hunk 1]
```diff
@@ -55,7 +55,21 @@ julia> uprint(pac, :block)
  ▀█████████████████ 
   ▀██████████████▀  
      ▀▀██████▀▀     
+
+julia> uprint(pac, :sextant)
+ 🬞🬵🬻██🬺🬱🬏
+🬞███🬕🬨███🬓
+🬻█████🬝🬆🬀
```
Comment (rafaqz) [inline README.md:62]: Why is so much detail lost here ... Shouldn't it be bigger?

File: test/references/octant_ghost.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+𜺠𜴯𜷥█𜴴𜷞𜺣
+𜷡🯦𜷕𜷤🯦𜷕▙
+█𜶫█🮅█𜵰█
```
Comment (rafaqz) [inline test/references/octant_ghost.txt:3]: These don't seem right. Block and braile have the same detail just different sizes, and are both complete representations of the array they're made from. But this has much less detail than the array does, so it's lossy

**Review by rafaqz — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (eschnett) [reply README.md:62]: This is what I see:
```
julia> uprint(pac, :sextant)
 🬞🬵🬻██🬺🬱🬏
🬞███🬕🬨███🬓
🬻█████🬝🬆🬀
█████🬲🬏  🬇
🬨██████🬺🬱🬏
 🬬██████🬝🬀
  🬁🬊🬎🬎🬆🬀
```
Well, my browser font doesn't seem to have sextants (my terminal font does), but it's six lines long, not just three.

Comment (rafaqz) [reply README.md:62]: Ah it's just browser fonts

File: test/references/octant_ghost.txt (hunk 2 above)
Comment (eschnett) [reply test/references/octant_ghost.txt:3]: Your font doesn't have octant characters. You are seeing some of the characters (the all-white and all-black ones), but it seems that the missing characters are replaced by spaces. I see a "missing character" symbol instead when I use the wrong font.

File: README.md (hunk 1 above)
Comment (eschnett) [reply README.md:62]: pac-sixel

File: test/references/octant_ghost.txt (hunk 2 above)
Comment (eschnett) [reply test/references/octant_ghost.txt:3]: pac-octant-missing
This is how missing characters look for me. Not ideal, but it's obvious that something is going wrong.

File: README.md (hunk 1 above)
Comment (rafaqz) [reply README.md:62]: Ah ok perfect!

Comment (eschnett) [reply README.md:62]: (I am using the [JuliaMono](https://juliamono.netlify.app) font.)

**Pull request merged**